api/routes/me/post.js

```diff
@@ -1,9 +0,0 @@
-const logger = require('../../logger')('me route post');
-
-module.exports = app => {
-  logger.debug('setting up POST endpoint');
-  app.post('/logout', (req, res) => {
-    res.clearCookie('gov.cms.eapd.api-token');
-    return res.status(200).end();
-  });
-};
```

Comment:
That's interesting. I wouldn't have thought to look for the route associated with `POST /logout` in `api/routes/me`